website/thaliawebsite/management/commands/dataminimisation.py

```diff
@@ -1,6 +1,7 @@
 from django.core.management.base import BaseCommand
 
-from members import services
+from members import services as members_services
+from events import services as events_services
 
 
 class Command(BaseCommand):
```

Comment:
If we want this command as general purpose minimisation command it should probably move out of the members app.

Reply:
It is in the `thaliawebsite` app